test/Microsoft.DotNet.CommandFactory.Tests/GivenALocalToolsCommandResolver.cs

```diff
@@ -124,6 +124,31 @@ public void WhenNuGetGlobalPackageLocationIsCleanedAfterRestoreItShowError()
                 toolCommandNameA.ToString()));
         }
 
+        [Fact]
+        public void WhenNuGetGlobalPackageLocationIsNotRestoredItShowError()
```

Comment:
NIT: maybe change the name to `WhenNuGetGlobalPackageLocationIsNotRestoredItThrowsGracefulException` since that's what you are testing?